gogettr/capabilities/registration_status.py

```diff
@@ -0,0 +1,18 @@
+from gogettr.capabilities.base import Capability
+
+
+class RegistrationStatus(Capability):
+    def is_registered(
+        self,
+        username: str,
+    ) -> bool:
+        """Checks if username is registered in Gettr
+
+        :param str username: username to check"""
+
+        url = f"/s/user/{username}/exists"
+
+        data = self.client.get(
+            url,
+        )
+        return data
```

Comment:
Perhaps we could insert a bit more data into the output (e.g., including the username itself)? This will be handy when the actual output is JSON(L). I propose we do something like this:

```suggestion
        return dict(username=username, registered=data)
```